src/components/ContactForm/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useRef, useState } from "react";
+import emailjs from "emailjs-com";
+import {
+  Container,
+  Form,
+  FormInput,
+  InputError,
+  RodoInformation,
+  Message,
+  SendButton,
+  ReCAPTCHAContainer,
+  NameInputContainer,
+  EmailInputContainer,
+  MessageInputContainer,
+  Label,
+} from "./style";
+import { useForm } from "react-hook-form";
+import ReCAPTCHA from "react-google-recaptcha";
+
+type FormValues = {
+  name: string;
+  email: string;
+  message: string;
+};
+
+const Contact = () => {
+  const [emailSent, setEmailSent] = useState(false);
+  const [isNotRobot, setIsNotRobot] = useState(false);
+  const captchaRef = useRef<ReCAPTCHA | null>(null);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    setValue,
+  } = useForm<FormValues>();
+  const onSubmit = handleSubmit((data) => {
+    const { name, email, message } = data;
+    console.log(data);
+    if (isNotRobot && name && email && message) {
+      const serviceId = process.env.REACT_APP_SERVICE_ID as string;
+      const templateId = process.env.REACT_APP_TEMPLATE_ID as string;
+      const publicKey = process.env.REACT_APP_EMAILJS_PUBLIC_KEY as string;
+      const templateParams = {
+        name,
+        email,
+        message,
+      };
+      emailjs
+        .send(serviceId, templateId, templateParams, publicKey)
+        .then((response) => {
+          setEmailSent(true);
+          console.log(response);
+          setValue("name", "");
+          setValue("email", "");
+          setValue("message", "");
+          if (captchaRef.current) {
+            captchaRef.current.reset();
+          }
+        })
+        .catch((error) => console.log(error));
+    }
+  });
+  const onChange = () => {
+    setIsNotRobot(true);
+  };
+
+  return (
+    <Container>
+      {emailSent ? (
+        <h1>
+          Thank you for your message, we write you back as soon as possible!
+        </h1>
+      ) : (
+        <>
+          <h1>Want to connect? Let's talk!</h1>
+          <Form onSubmit={onSubmit}>
+            <NameInputContainer>
+              <Label>Name</Label>
```

Comment:
When you use `label` next to `input` you should add `htmlFor` on `label` and `id` on `input` - thanks to that screen reader will know what is the label for focused input.

Reply:
you can read more: https://www.w3.org/WAI/tutorials/forms/labels/